src/notation/qml/MuseScore/NotationScene/internal/EditStyle/ChordSymbolsPage.qml

```diff
@@ -198,24 +198,41 @@ StyledFlickable {
                 }
 
                 StyledGroupBox {
+                    Layout.fillWidth: true
+                    title: qsTrc("notation/editstyle/chordsymbols", "Capitalization")
 
-                    label: CheckBox {
-                        id: capitaliseCheckBox
-                        text: qsTrc("notation/editstyle/chordsymbols", "Automatically capitalize note names")
+                    ColumnLayout {
+                        spacing: 12
+                        Layout.fillWidth: true
 
-                        checked: chordSymbolsModel.automaticCapitalization && Boolean(chordSymbolsModel.automaticCapitalization.value)
-                        onClicked: {
-                            if (chordSymbolsModel.automaticCapitalization) {
-                                chordSymbolsModel.automaticCapitalization.value = !checked
-                            }
-                        }
-                    }
+                        RowLayout {
+                            spacing: 6
+                            Layout.fillWidth: true
 
-                    Layout.fillWidth: true
+                            Item {
```

Comment:
Why did you wrap this checkbox in an `Item`?

Reply:
You can also set `Layout.preferredWidth` directly on the CheckBox. I see that it's indeed also done that way further down in the file, but it shouldn't be necessary there either.